challenge.md

```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git clone url
 
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git status
 
 <!-- Answer Ends Here -->
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git commit --amend
 
 <!-- Answer Ends Here -->
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->origin...not sure.
 
 <!-- Answer Ends Here -->
 
 5. Suppose that we wanted to look at the third-to-last commit on `master`; what command(s) would we write to do that?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->git reset --hard HEAD~3
```

Comment:
Oh no - please don't do that! `git reset --hard` resets both Git and your actual files to a prior state, **permanently**. 

If we just want to look at a previous commit, we can use `git checkout`.

Also, the third-to-last commit is `HEAD~2`, not `HEAD~3`.